jobs/ingestor_cloudwatch/templates/config/input_and_output.conf.erb

```diff
@@ -12,6 +12,23 @@ input
   }
 }
 
+filter
+{
+    mutate {
+        rename => {"[cloudwatch_logs][tags][environment]"=>"environment"}
+        rename => {"[cloudwatch_logs][tags][OrganizationGUID]"=>"[@cf][org_id]"}
+        rename => {"[cloudwatch_logs][tags][SpaceGUID]"=>"[@cf][space_id]"}
+        rename => {"[cloudwatch_logs][tags][Organizationname]"=>"[@cf][org]"}
+        remove_field => ["[cloudwatch_logs][tags][Createdat]"]
+        remove_field => ["[cloudwatch_logs][tags][Updatedat]"]
+        rename => {"[cloudwatch_logs][tags][InstanceGUID]"=>"service_instance_id"}
+        rename => {"[cloudwatch_logs][tags][Serviceofferingname]"=>"service_offering"}
+        rename => {"[cloudwatch_logs][tags][Serviceplanname]"=>"service_plan"}
```

Comment:
```suggestion
        rename => {"[cloudwatch_logs][tags][InstanceGUID]"=>"[@cf][service_instance_id]"}
        rename => {"[cloudwatch_logs][tags][Serviceofferingname]"=>"[@cf][service_offering]"}
        rename => {"[cloudwatch_logs][tags][Serviceplanname]"=>"[@cf][service_plan]"}
```

Since these are CF specific values, i thought maybe they should go in that field group